litellm/integrations/keywordsai.py

```diff
@@ -0,0 +1,145 @@
+from litellm.utils import ModelResponse
+from litellm import stream_chunk_builder, print_verbose
+from litellm.integrations.custom_logger import CustomLogger
+import requests
+import os
+import json
+import datetime
+
+class BgColors:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKCYAN = '\033[96m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+KEYWORDS_AI_API_KEY = os.getenv("KEYWORDS_AI_API_KEY")
+def setup_keywordsai_params(model, messages, kwargs):
+    url = "https://api.keywordsai.co/api/request-logs/create/"
+
+    if not KEYWORDS_AI_API_KEY:
+        print(f"{BgColors.WARNING}!!!!WARNING: KEYWORDS_AI_API_KEY not found in environment!!!!{BgColors.ENDC}")
+    headers = {
+        "Authorization": f"Bearer {KEYWORDS_AI_API_KEY}",
+        "Content-Type": "application/json",
+    }
+
+    keywordsai_params = kwargs.get("extra_body", {}).pop("keywordsai_params", {})
```

Comment:
this looks wrong. you should be taking this information from metadata. extra_body will be sent straight to the llm provider.

Reply:
Hey, I think I meant to do this because we support an LLM API call to api.keywordsai.co use extra_body.

I will change it to metadata per your request.

Reply:
Fixed